commercial-schedules/src/main/java/org/dcsa/conformance/standards/cs/party/CsPublisher.java

```diff
@@ -0,0 +1,138 @@
+package org.dcsa.conformance.standards.cs.party;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.*;
+import java.util.function.Consumer;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+import lombok.extern.slf4j.Slf4j;
+import org.dcsa.conformance.core.party.ConformanceParty;
+import org.dcsa.conformance.core.party.CounterpartConfiguration;
+import org.dcsa.conformance.core.party.PartyConfiguration;
+import org.dcsa.conformance.core.party.PartyWebClient;
+import org.dcsa.conformance.core.scenario.ConformanceAction;
+import org.dcsa.conformance.core.state.JsonNodeMap;
+import org.dcsa.conformance.core.toolkit.JsonToolkit;
+import org.dcsa.conformance.core.traffic.ConformanceMessageBody;
+import org.dcsa.conformance.core.traffic.ConformanceRequest;
+import org.dcsa.conformance.core.traffic.ConformanceResponse;
+import org.dcsa.conformance.standards.cs.action.SupplyScenarioParametersAction;
+import org.dcsa.conformance.standards.cs.model.CsDateUtils;
+
+@Slf4j
+public class CsPublisher extends ConformanceParty {
+
+  public CsPublisher(
+      String apiVersion,
+      PartyConfiguration partyConfiguration,
+      CounterpartConfiguration counterpartConfiguration,
+      JsonNodeMap persistentMap,
+      PartyWebClient webClient,
+      Map<String, ? extends Collection<String>> orchestratorAuthHeader) {
+    super(
+        apiVersion,
+        partyConfiguration,
+        counterpartConfiguration,
+        persistentMap,
+        webClient,
+        orchestratorAuthHeader);
+  }
+
+  @Override
+  protected void exportPartyJsonState(ObjectNode targetObjectNode) {}
```

Comment:
Can you add a comment why this 2 methods can be empty? They also don't throw anything.. not sure if that is needed anyways.